macOS/DuckDuckGo/VisualRefresh/VisualStyleConfigurable.swift

```diff
@@ -21,18 +21,88 @@ import BrowserServicesKit
 import FeatureFlags
 
 protocol VisualStyleConfigurable {
```

Comment:
Sorry for not raising it in your previous pull request, but I’d like to revise the naming here. The name `VisualStyleConfigurable` indicates a trait of a view (e.g. you could make NSButton “visual style configurable” by conforming to that protocol). Meanwhile, this protocol actually defines the API for `VisualStyleManager`, which only provides styling values for views.

What would you say if we did the following:
* rename `VisualStyleConfigurable` to `VisualStyleProviding`,
* rename `VisualStyleManager` to `VisualStyleProvider`.
* rename `visualStyleConfigurable` properties all over the codebase as `visualStyleProvider` or just `visualStyle`.

Reply:
Good call. Changed!